src/components/Datepicker/sgds-datepicker.scss

```diff
@@ -0,0 +1,33 @@
+@import "./sgds-datepicker-calendar.scss";
+@import "./sgds-datepicker-header.scss";
+
```

Comment:
--datepicker-theme-color is currently setting the colors for the calendar dates, months and years. 

but it is not controlling the hovering colour, and button color. so if a user sets 

--datepicker-theme-color : orange 

they will also need to set the following to sync all up to a orange theme 

--datepicker-hover-background-color : "another shade of orange" 
--datepicker-closebutton-background-color: "orange" 
--datepicker-closebutton-hover-background-color: "another darker shade of orange" 

@rav3 thats the reason why im thinking of a way to calculate the shadings instead. Was trying out string interpolations to generate --sgds-primary-100, -sgds-primary-600 from --sgds-primary but the scss interpolations cannot work with css custom variables and seems like there is no way to interpolate css custom variables,

 an alternative will be to use sass functions/mixin like the shade() and tint() to calculate out the colors for -100 and -600 . but from what petrine mention, i think she is not using shade() or tint() to calculate the colors. Also dont think there will be one function that will be able to calculate all the shades, since some colours were manually set.

Two resolutions i have in mind: 

1) So either we give up using --sgds-primary-100, --sgds-primary-600 (#491db6) here and calculate a close enough shade to it derived from  --datepicker-theme-color: #5925DC. In this case users just define theme-color once and all other parts of datepicker will follow that theme 

2) or we just let the users manually change all the colours themselves. Then preferably dont call --datepicker-theme-color anymore , can rename it to --datepicker-calendar-color to avoid confusion. 
In this case, we dont expect them to follow a single colour rule for the datepicker. they can go creative with diff colors lol 

any suggestions or ideas to resolve this? @haphizi @rav3

Reply:
think other components like stepper also similar case

Reply:
I feel option 1 would be more convenient for the users, as they just need to set 1 custom variable and the rest will fall in place. Not exactly sure if users would want disco colors but i assume most would follow the same standardized color themes throughout.